src/ocad-to-svg.js

```diff
@@ -241,16 +241,14 @@ const objectToSvg = (options, symbols, object) => {
             node.children = node.children.concat([
               lineToPath(
                 object.coordinates,
-                dbl.dblLeftWidth * 1.5 +
-                  dbl.dblRightWidth * 1.5 +
-                  dbl.dblWidth * 2,
```

Comment:
I think I experimented with these to get the Mapbox rendering to look correct, but I realize the SVG rendering works without them, so it's probably some peculiarity with Mapbox and shouldn't be part of the reader. :+1: